registrations/models.py

```diff
@@ -146,6 +146,36 @@ def fire_language_metric(sender, instance, created, **kwargs):
         })
 
 
+@receiver(post_save, sender=Registration)
+def fire_source_metric(sender, instance, created, **kwargs):
+    """
+    Fires metrics for each source for each subscription, a sum metric for
+    the registrations over time, and a last metric for the total count.
+    """
+    from .tasks import fire_metric
+    if (created):
+        if "hw" in instance.source.authority:
+            source = 'hcw'
+            sources = ['hw_limited', 'hw_full']
+        else:
+            source = 'public'
+            sources = ['patient', 'advisor']
```

Comment:
I'm wondering if it would be better to just keep it as `hw_limited`, `patient`, etc. That way if later on they wanted to see the split between patient and advisor, they could, and getting the total `public` like they want now would be just adding two numbers. I think that's the way that we've implemented this for hellomama. Is this a strict requirement on the ticket?